tests/test_2048.py

```diff
@@ -1,4 +1,60 @@
-import pytest
-import py_cui_2048.game as GAME
+""" Test
+    """
 
-#TODO
\ No newline at end of file
+
+def test_game_board_positions(game_instance):
+    """test `game_board` positions"""
+
+    # _                                            [y][x]
+    assert game_instance.game_board.board_positions[1][0] == 2
+
+    assert game_instance.game_board.board_positions[3][2] == 2
+
+    assert game_instance.game_board.get_empty_pos_list() == [
+        [0, 0],
+        [0, 1],
+        [0, 2],
+        [0, 3],
+        [1, 1],
+        [1, 2],
+        [1, 3],
+        [2, 0],
+        [2, 1],
+        [2, 2],
+        [2, 3],
+        [3, 0],
+        [3, 1],
+        [3, 3],
+    ]
+
+    random_tile_success = game_instance.game_board.add_random_tile()
+
+    while random_tile_success is False:
+        random_tile_success = game_instance.game_board.add_random_tile()
+
+    assert len(game_instance.game_board.get_empty_pos_list()) < 14
+
+
+def test_check_victory(game_instance):
+    """test `check_victory` method"""
+
+    assert game_instance.check_victory() is None
```

Comment:
This is actually catching unintended behavior - I would think `check_victory` should return `False` if victory is not found. Could you change the underlying method to do this and update this test case?

Reply:
i thought the same.

Reply:
After merge i would have to get the program running in my environment to do some integration tests

Reply:
I Just don't want to make any breaking changes to your code just yet